doc/classes/AudioStreamPlayer.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		Plays an audio stream non-positionally.
 		To play audio positionally, use [AudioStreamPlayer2D] or [AudioStreamPlayer3D] instead of [AudioStreamPlayer].
+		However, if you want manual control of stereo (left/right) volumes, this will work.
```

Comment:
```suggestion
		To play audio positionally, use [AudioStreamPlayer2D] or [AudioStreamPlayer3D] instead of [AudioStreamPlayer]. [AudioStreamPlayer] only provides manual stereo panning using the [member volume_scale_l] and [member volume_scale_r] properties.
```

Reply:
Applied in latest commit.